Add acknowledgements for eslint-plugin-package-json and eslint-plugin-depend (#19)
prefer-jsr/prefer-jsr: README.md

4. No type errors: `pnpm exec nx run-many -t typecheck`
5. Use conventional commits for your commit messages

## Acknowledgements

> This project was inspired by and draws ideas from other ESLint plugins that help maintain quality in `package.json` files:

Reviewer: ```suggestion
The lint plugin in this project was inspired by and draws ideas from other ESLint plugins that help maintain quality in `package.json` files:
```

- [eslint-plugin-package-json](https://github.com/zetlen/eslint-plugin-package-json) - ESLint rules for package.json files

> - [eslint-plugin-depend](https://github.com/es-tooling/eslint-plugin-depend) - ESLint plugin for managing dependencies

Reviewer: ```suggestion
- [eslint-plugin-depend](https://github.com/es-tooling/eslint-plugin-depend)
```

> We're grateful to these projects for paving the way and demonstrating effective patterns for linting package.json files.

Reviewer: ```suggestion
Thank you to these projects for leading the way and demonstrating effective patterns for linting package.json files.
```
